instance/tests/integration/base.py

```diff
@@ -59,7 +59,7 @@ def tearDown(self):
         for load_balancer in LoadBalancingServer.objects.iterator():
             load_balancer.delete()
         for instance in OpenEdXInstance.objects.iterator():
-            instance.delete()
+            instance.delete(ignore_errors=True)
```

Comment:
Why are you passing ignore errors to the function?

Reply:
The `ignore_errors` attribute was introduced recently.
I thought it would be a good idea since we're deleting an instance and if there's an error with deleting we won't go all the way through the deletion. For instance, if there's an error while deleting a rabbitMQ user the entire deletion workflow will stop there and throw the error.
This will log those errors and continue with deletion/deprovisioning the rest of the stuff.

Reply:
Nice :+1: